srtcore/atomic_msvc.h

```diff
@@ -0,0 +1,68 @@
+/*
+* Copyright © 2018, VideoLAN and dav1d authors
+* All rights reserved.
+*
+* Redistribution and use in source and binary forms, with or without
+* modification, are permitted provided that the following conditions are met:
+*
+* 1. Redistributions of source code must retain the above copyright notice, this
+*    list of conditions and the following disclaimer.
+*
+* 2. Redistributions in binary form must reproduce the above copyright notice,
+*    this list of conditions and the following disclaimer in the documentation
+*    and/or other materials provided with the distribution.
```

Comment:
Worth checking what do we have to do.
Would we need to put "Copyright © 2018, VideoLAN" somewhere else, e.g. in our [LICENCE](https://github.com/Haivision/srt/blob/master/LICENSE).